src/Stack.re

```diff
@@ -266,17 +262,23 @@ module Make = (M: {type params;}) => {
     headerMode: option(string),
     keyboardHandlingEnabled: option(bool),
   };
-
+  type renderCallbackProp = {
+    navigation,
+    route,
+  };
   type screenProps('params) = {
     name: string,
     options: option(optionCallback),
     initialParams: option('params),
     component:
-      React.component({
-        .
-        "navigation": navigation,
-        "route": route,
-      }),
+      option(
+        React.component({
+          .
+          "navigation": navigation,
+          "route": route,
+        }),
+      ),
+    children: option(renderCallbackProp => React.element),
```

Comment:
Maybe instead we should have screenProps & screenPropsWithCallback?

Reply:
https://github.com/reason-react-native/reason-react-navigation/blob/master/src/Stack.re#L288

But Screen here can accept only one type right?